src/components/TaskCard/TaskCard.jsx

```diff
@@ -1,29 +1,49 @@
 import { useDispatch } from "react-redux";
 import PropTypes from "prop-types";
-import { deleteTask } from "src/store/task/actions/taskActions";
-import DeleteButton from "src/components/Buttons/DeleteButton/DeleteButton";
+import { deleteTask, markTaskDone } from "src/store/task/actions/taskActions";
+import { formatDate } from "src/utils/formatDate";
+import TaskCardButtonsContainer from "../TaskCardButtonsContainer/TaskCardButtonsContainer";
 import "./TaskCard.scss";
 
-function TaskCard({ taskId, taskTitle, createdAt }) {
+function TaskCard({ taskId, taskTitle, createdAt, taskDoneAt, isTaskDone }) {
   const dispatch = useDispatch();
 
   function handleDeleteTask() {
     dispatch(deleteTask(taskId));
   }
 
+  function handleTaskDone() {
+    dispatch(markTaskDone(taskId));
+  }
+
   return (
     <div className="tasks-container__box">
-      <h3>{taskTitle}</h3>
-      <p>Created At: {createdAt}</p>
-      <DeleteButton onDelete={handleDeleteTask} />
+      <h3 className={`${isTaskDone ? "task_done" : ""}`}>{taskTitle}</h3>
```

Comment:
Please have a check if that also works.
```suggestion
      <h3 className={`${isTaskDone && "task_done" }`}>{taskTitle}</h3>
```